src/components/05_pages/NodeForm/index.js

```diff
@@ -29,6 +29,11 @@ class NodeForm extends React.Component {
       PropTypes.oneOfType([
         PropTypes.func,
         PropTypes.instanceOf(React.Component),
+        PropTypes.shape({
```

Comment:
I'm a bit confused. I would have expected it would be a ```PropTypes.oneOfType([PropTypes.func, PropTypes.instanceOf(React.Component), PropTypes.shape({component: PropTypes.instanceOf(React.component), multiple: PropTypes.booln})])```

Reply:
@dawehner I also thought of this but `component` is available inside `options_select`. That's why implemented it in this way.

Reply:
Looking at https://github.com/jsdrupal/drupal-admin-ui/blob/master/src/components/05_pages/NodeForm/Widgets.js#L11 it looks like all of them have a component now :)

Reply:
Yeah you are right here, that's why we need to implement this part of code which I implemented for `options_select` for other widgets too. And @mattgrill suggested that we can cover that in follow up ticket. So, for now I did for `options_select` only.